libeos-updater-util/flatpak.c

```diff
@@ -1578,6 +1578,9 @@ format_remote_ref_action_type (EuuFlatpakRemoteRefActionType action_type)
   return enum_value->value_nick;
 }
 
+/* NOTE: The output produced by this function is parsed by
+ * eos-updater-prepare-volume, so be mindful of that if you make any changes!
+ */
```

Comment:
It might be better to dump JSON in this case :)

Reply:
On the one hand that's added complexity and one more thing to maintain, but I suppose it's worth it to have something machine readable and a bit more robust.